src/main/java/pl/most/typer/model/matches/TeamGoals.java

```diff
@@ -18,4 +16,8 @@
     private Integer id;
     private Integer homeTeam;
     private Integer awayTeam;
+
+    @ManyToOne
```

Comment:
Czy ten sam obiekt TeamGoals moze byc przypisany to róznych obiektow Score?

Reply:
Wstepnie tak miało być, ze ten sam obiekt może być w rożnych obiektach score, ale za dużo sztukowania w tym było i finalnie nie ogarnąłem :p Zmieniłem na OneToOne